monkey/infection_monkey/utils/aws_environment_check.py

```diff
@@ -0,0 +1,27 @@
+import logging
+
+from common.cloud.aws.aws_instance import AwsInstance
+from infection_monkey.telemetry.aws_instance_telem import AwsInstanceTelemetry
+
+logger = logging.getLogger(__name__)
+
+_AWS_ENVIRONMENT = "AwsEnvironment"
+
+
+def _running_on_aws(aws_instance: AwsInstance) -> bool:
+    return aws_instance.is_instance()
+
+
+def report_aws_environment():
+    logger.info("Collecting AWS info")
+    info = {_AWS_ENVIRONMENT: {}}
+
+    aws_instance = AwsInstance()
+
+    if _running_on_aws(aws_instance):
+        logger.info("Machine is an AWS instance")
+        info[_AWS_ENVIRONMENT]["instance_id"] = aws_instance.get_instance_id()
```

Comment:
What's the point in crafting `{_AWS_ENVIRONMENT: { "instance_id": instance_id}}`? Why can't we just AwsInstanceTelemetry({"instance_id": instance_id}).send()